app/src/main/java/mozilla/lockbox/presenter/ItemDetailPresenter.kt

```diff
@@ -16,13 +18,32 @@ import mozilla.lockbox.store.DataStore
 interface ItemDetailView {
     var itemId: String?
     fun updateItem(item: ItemDetailViewModel)
+
+    val addressLayoutClicks: Observable<Unit>
+    fun openWebsite(hostname: String)
 }
 
 class ItemDetailPresenter(
     private val view: ItemDetailView,
     private val dispatcher: Dispatcher = Dispatcher.shared,
     private val dataStore: DataStore = DataStore.shared
 ) : Presenter() {
+
+    override fun onViewReady() {
+        this.view.addressLayoutClicks
+                .subscribe {
+                    view.itemId?.let {
+                        dataStore.get(it)
+                                .subscribe {
+                                    Log.d("hello", "${it!!.hostname}, ${it!!.formSubmitURL}")
```

Comment:
Is this ready for review yet? This should be marked `WIP` until debugging statements are cleaned up.

Reply:
yes is ready, I had removed forgotten log entry